nservicebus/operations/metrics_reporting_metrics_[1.0,1.1).partial.md

```diff
@@ -0,0 +1,29 @@
+## Reporting metrics data
```

Comment:
The are obsoleted from 1.1 which is not described in these partial. Should I copy this to `[1.1,)` and mark as obsoleted? If yes, how should I mention the preferred way of reporting with observers? Mention it first, then say, that all the methods below are obsoleted now?